### DISCUSSION TITLE
Strapi doesn't build under 1 gig ram, is there any fix to this ?

### DISCUSSION THREAD

**Discussion by fellz** [strapi/strapi#8167] [Legacy Topics]
Hi
I am trying to run site on prod server but it appears that 1 gig of ram is not enough. Is there any fix to this ? may be some webpack tweaks ?

**Comment by derrickmehaffy** [comment]
We are planning to remove the GitHub discussions in early 2021, in the future can you please ask your questions on our [Forum](https://forum.strapi.io/). As your question is easy to answer, I'll go ahead and do so.

Currently no, the admin panel build process requires quite a bit of ram, the minimum required is around 2GB. We have made many modifications to try and cut this, but most of the issue revolves around the sharp library to optimize images and do some minimal editing of uploaded images.

For the moment we don't have a work around other than building the admin locally and committing the `build` folder into your source control.
